chi2/chinotype.py

```diff
@@ -593,14 +670,15 @@ def dbtrx():
 
 
     def getFilterSql(self):
-        sql = 'select c_name from {0}.schemes'.format(self.metaschema)
+        sql = 'select c_name from {0}'.format(self.chischemes)
```

Comment:
Changed this to operate on `chischemes` (see changes starting with line 453) instead of `SCHEMES`